### PR TITLE
Update index.d.ts

### PR DESCRIPTION
**Pull request by rapomon**
The message response can be an Array or an string since the last version.
I'm the creator of the original library.

Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/pwd-strength/index.d.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ interface PwdStrengthOptions {
 interface PwdStrengthReturn {
     success: boolean;
     key: string;
-    message: string;
+    message: Array<string> | string;
```
Comment (MysteryBlokHed) [inline types/pwd-strength/index.d.ts:45]: The CI error should be fixed by changing this type.

```suggestion
    message: string[] | string;
```

Comment (rapomon) [reply types/pwd-strength/index.d.ts:45]: Thank you, this change will work.

You can check the documentation here (string or string[] case):
https://github.com/rapomon/pwd-strength/#error-response
https://github.com/rapomon/pwd-strength/#error-response-with-allerrors-true-setting

Sorry but I don't have time to do a unit test for this change or follow the DefinitelyTyped rules for this change to be  published.

**Review by himynameisdave — APPROVED** [review]
Approving, but you'll still need to update this to use the `string[]` shorthand.

**Comment by rapomon** [comment]
> Approving, but you'll still need to update this to use the `string[]` shorthand.

Ok, thank oyu.

**Comment by himynameisdave** [comment]
Not sure how to do this, but @rapomon we should probably add you as a codeowner [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6870fcb428850b1b0e90404ebc4a7ca6ab6b3184/.github/CODEOWNERS#L5701), since you are the library author. It's probably automated somehow.

**Comment by MysteryBlokHed** [comment]
> Not sure how to do this, but @rapomon we should probably add you as a codeowner [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6870fcb428850b1b0e90404ebc4a7ca6ab6b3184/.github/CODEOWNERS#L5701), since you are the library author. It's probably automated somehow.

If his name is added to the "Definitions by" part of the file header, he'd be added as a codeowner automatically.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6870fcb428850b1b0e90404ebc4a7ca6ab6b3184/types/pwd-strength/index.d.ts#L3

**Comment by rapomon** [comment]
> 
Yes, @himynameisdave , maybe I should be added as the codeowner in order to do future changes. Thank you.

**Pull request closed without merging**